### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by macmv**
Ticket(s): ENG-5576

Updates README for all the existing commands, and adds docs for `fauna project` and `fauna stack`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,33 +96,33 @@ my_second_app
...
 This is how you create a key for the database `my_app`:
 
-```sh-session
+```sh
 $ fauna create-key my_app
 creating key for database 'my_app' with role 'admin'
 
 created key for database 'my_app' with role 'admin'.
-secret: ****************************************
+secret: fnAFPULk2WAAQY9t4x0tduzuz85gC-suDbTnl7um # this will be different
```
Comment (jdcryans) [inline README.md:116]: I'd prefer keeping the original or have something like <your key here>

File: README.md [hunk 2]
```diff
@@ -9,48 +9,54 @@
...
+```sh
 $ fauna cloud-login
 ```
 
-You will be prompted for your `email` and `password` from your [Fauna](https://dashboard.fauna.com) account.
+You will be prompted for your `email` and `password` from your
+[Fauna](https://dashboard.fauna.com) account.
 
-If you would like to use 3rd party identity providers like Github or Netlify, please refer to [this guide](https://docs.fauna.com/fauna/current/start/cloud-github.html).
+If you would like to use 3rd party identity providers like Github or Netlify,
```
Comment (jdcryans) [inline README.md:50]: Remove this section, also that link is broken.

File: README.md [hunk 3]
```diff
@@ -113,7 +134,7 @@ Key ID               Database             Role
 
 And then delete the key with id: `200219702370238976`:
 
-```sh-session
+```sh
```
Comment (jdcryans) [inline README.md:137]: I just noticed a few lines below that the header #command doesn't work anymore, it's list-of-commands

File: README.md [hunk 4]
```diff
@@ -211,7 +225,7 @@ my_app> Post.create({ title: "What I had for breakfast .." })
 
 We can also insert items in bulk by using iterator functions on arrays.
 
-```javascript
+```ts
 my_app> ["My cat and other marvels", "Pondering during a commute", "Deep meanings in a latte"].map(title => Post.create({ title: title }))
```
Comment (jdcryans) [inline README.md:229]: This line is too long and gets clipped in GH, mind fixing it while were here?

File: README.md [hunk 5]
```diff
@@ -277,155 +291,137 @@ my_app> Post.byId("373143473418666496")!.replace({ title: "My dog and other marv
...
-scheme=https
-secret=OTHER_FAUNA_SECRET
-graphqlHost=cluster-us-east.example.com
-graphqlPort=443
 ```
 
 # Connecting to local endpoints
 
-If you are running Fauna locally using our Docker images, you may need to configure the Shell to work with local endpoints so you can interact with the databases running in the Docker containers.
+If you are running Fauna locally using our Docker image, you will need to
```
Comment (jdcryans) [inline README.md:358]: Seems like a good place to add a link to https://docs.fauna.com/fauna/current/tools/dev

File: README.md [hunk 6]
```diff
@@ -454,77 +451,46 @@ Post.create({
...
-DESCRIPTION
-  Adds a connection endpoint for Fauna.
-
-EXAMPLE
-  $ fauna add-endpoint https://db.fauna.com:443
-  $ fauna add-endpoint http://localhost:8443/ --alias localhost --key secret
-```
-
-_See code: [src/commands/add-endpoint.js](src/commands/add-endpoint.js)_
+- [`fauna autocomplete`](#fauna-autocomplete)
```
Comment (jdcryans) [inline README.md:472]: A lot of those links are broken.

File: src/commands/add-endpoint.ts [hunk 7]
```diff
@@ -99,6 +99,8 @@ Adds an endpoint to ~/.fauna-shell.
       if (res.status !== 200) {
         ux.action.stop();
         console.log("Warning: invalid secret");
+      } else {
```
[the remark this answers is not in the document]
Comment (macmv) [reply src/commands/add-endpoint.ts:102]: This was a breaking change from a previous PR, it was just easier to fix it in here.

File: README.md (hunk 1 above)
Comment (jdcryans) [reply README.md:112]: I totally a few words. Oh so the behavior used to be that we'd hid the secret and you aligned it?

Comment (jdcryans) [reply README.md:112]: He says yes.

**Review by jdcryans — APPROVED** [review]

**Pull request merged**